zerver/views/messages.py

```diff
@@ -555,15 +555,16 @@ def exclude_muting_conditions(user_profile: UserProfile,
     return conditions
 
 @has_request_variables
-def get_messages_backend(request, user_profile,
-                         anchor = REQ(converter=int),
-                         num_before = REQ(converter=to_non_negative_int),
-                         num_after = REQ(converter=to_non_negative_int),
-                         narrow = REQ('narrow', converter=narrow_parameter, default=None),
-                         use_first_unread_anchor = REQ(validator=check_bool, default=False),
-                         client_gravatar = REQ(validator=check_bool, default=False),
-                         apply_markdown = REQ(validator=check_bool, default=True)):
-    # type: (HttpRequest, UserProfile, int, int, int, Optional[List[Dict[str, Any]]], bool, bool, bool) -> HttpResponse
+def get_messages_backend(request: HttpRequest,
+                         user_profile: UserProfile,
```

Comment:
Minor nitpick but if possible, I think it would be better to maintain the original arrangement of arguments as much as possible. So if it is possible to keep it under the line limit, we should have `request` and `user_profile` in the same line.